apps/platform/utils/auth/passkeys.ts

```diff
@@ -115,12 +116,10 @@ async function generateAuthenticationOptions({
 
 async function verifyAuthenticationResponse({
   authenticationResponse,
-  authChallengeId,
-  expectedAllowedCredentials
+  authChallengeId
 }: {
   authenticationResponse: AuthenticationResponseJSON;
   authChallengeId: string;
-  expectedAllowedCredentials?: any;
```

Comment:
we should revert this!

expectedCrendtials is used when we know the user that wants to login/verify - so we pass an array of acceptable authenticators that the UI will show

its used often for "re-auth" to verify the person taking the action is indeed the user (delete an org, set up billing etc...)

not used yet, but will be soon

Reply:
sure, I will re-add it. I thought you left it out.